app_server/posts/models.py

```diff
@@ -46,6 +47,14 @@ def get_absolute_url(self):
         return reverse("post_detail", args=[self.id])
 
 
+class TagDescription(models.Model):
+    tag = models.ForeignKey("taggit.Tag", on_delete=models.CASCADE, related_name="tag_description")
+    description = models.CharField(max_length=200, blank=True, unique=False)
```

Comment:
I think we should have unique values here.

Reply:
ForeignKey already appoints the unique value to the db column. Please carry on with the merge.